### PR TITLE
[Feature][Merv] Add filter picker component

### PR DESCRIPTION
**Pull request by jdesolate**
- Added filter picker component 
- Added functionality

### CODE REVIEW

File: components/FilterPicker/index.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    transition: 'background-color 150ms ease',
+
+    width: 200,
+  },
+
+  icon: {
+    transform: opened ? 'rotate(180deg)' : 'rotate(0deg)',
+    transition: 'transform 150ms ease',
+  },
+
```
Comment (tapsTapayan) [inline components/FilterPicker/index.tsx]: Migrate to stylesheet

File: components/Search/index.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { ActionIcon, TextInput, useMantineColorScheme, useMantineTheme } from '@mantine/core';
+import { IconArrowLeft, IconArrowRight, IconSearch } from '@tabler/icons';
+import { ChangeEvent, useState } from 'react';
+import { usePropertyContext } from '../../hooks/PropertyContext';
+import style from "./_index.module.scss";
+
+function Search() {
+  const { filterProperties, properties, refreshProperties } = usePropertyContext();
+  const { colorScheme } = useMantineColorScheme();
+  const theme = useMantineTheme();
```
Comment (tapsTapayan) [inline components/Search/index.tsx]: use only colorscheme

File: components/Search/index.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          radius="md" 
+          color={theme.primaryColor} 
+          variant="filled" 
+          onClick={handleSearch}
+        >
+          {theme.dir === 'ltr' ? (
+            <IconArrowRight size={18} stroke={2} />
+          ) : (
+            <IconArrowLeft size={18} stroke={2} />
+          )}
```
Comment (tapsTapayan) [inline components/Search/index.tsx]: Place outside the return statement as a single element.

**Review by tapsTapayan — CHANGES_REQUESTED** [review]
edit lang @jdesolate

File: components/FilterPicker/index.tsx (hunk 1 above)
Comment (jdesolate) [reply components/FilterPicker/index.tsx]: resolved

File: components/Search/index.tsx (hunk 2 above)
Comment (jdesolate) [reply components/Search/index.tsx]: resolved in the last pr

File: components/Search/index.tsx (hunk 3 above)
Comment (jdesolate) [reply components/Search/index.tsx]: resolved in the last pr

**Review by tapsTapayan — APPROVED** [review]
approved @jdesolate

**Pull request merged**